internal/cmd/egctl/testdata/translate/out/rejected-http-route.route.yaml

```diff
@@ -79,8 +79,9 @@ httpRoutes:
         status: "False"
         type: Accepted
       - lastTransitionTime: null
-        message: Service envoy-gateway-system/backend not found
-        reason: BackendNotFound
+        message: 'Failed to process BackendRef 0 for the route rule 0: service envoy-gateway-system/backend
+          not found.'
+        reason: InvalidBackendRefs
```

Comment:
The current reason isn't specific enough. I plan to improve it in follow-up PRs to keep this one focused -- it's already quit large.

Reply:
the previous reason was a standard reason https://github.com/kubernetes-sigs/gateway-api/blob/703048161530940bcd3425bc51f21d034b4b6880/apis/v1beta1/shared_types.go#L143

Reply:
Yeah, I'm fixing this one.